x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/table/add_data_popover.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiBadge,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiPopover,
+  EuiPopoverFooter,
+  EuiPopoverTitle,
+  EuiText,
+} from '@elastic/eui';
+import {
+  ObservabilityOnboardingLocatorParams,
+  OBSERVABILITY_ONBOARDING_LOCATOR,
+} from '@kbn/deeplinks-observability';
+import { i18n } from '@kbn/i18n';
+import React, { useState } from 'react';
+import { useKibanaContextForPlugin } from '../../../../../hooks/use_kibana';
+
+export const AddDataPopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
```

Comment:
Nit: We can use the `useBoolean` hook like in the [Popover component](pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/common/popover.tsx/#L13-L14)